### ISSUE TITLE
Set the size of the validation set

### ISSUE THREAD

**Issue by maree-s3596596** [maree-s3596596/arff2pat#1]
Ability to set the size of the validation set.

Currently it is whatever percentage (float value [0.0,1.0]) set for the size of the test set applied again to the split training set to split the train into train and validation.

Add another option to set the validationsize (float value).

**Comment by maree-s3596596** [comment]
Added with commit 19189874f79bf21a2019286d3e59c06cbe9d3c3a
